packages/palette/src/elements/Skip/Skip.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import styled from "styled-components"
+import { Clickable, ClickableProps } from "../Clickable"
+import { Text } from "../Text"
+import { visuallyHiddenMixin, visuallyUnhiddenMixin } from "../VisuallyHidden"
+
+const Container = styled(Clickable)`
+  display: block;
+  text-decoration: underline;
+  ${visuallyHiddenMixin}
+
+  &:focus {
+    outline: 0;
+    ${visuallyUnhiddenMixin}
+  }
+`
+
+/** SkipProps */
+export type SkipProps = ClickableProps
+
+/** Skip */
+export const Skip = React.forwardRef(
+  (
+    { children, ...rest }: SkipProps,
+    forwardedRef: React.MutableRefObject<HTMLButtonElement>
+  ) => {
+    return (
+      <Container
+        ref={forwardedRef as any}
+        display="block"
+        bg="black10"
+        {...rest}
+      >
+        <Text p={1} variant="text">
+          {children}
+        </Text>
+      </Container>
+    )
+  }
+)
```

Comment:
This exports our skip link that is also used in the NavBar.

Reply:
Ah, I see -- mind moving this github comment into an actual docblock comment?